docs/secret_management.md

```diff
@@ -192,68 +164,55 @@ metadata:
     credentials:
       apiKey: ENC[k8s_secret@default/test-secret/api_key]
       appKey: ENC[k8s_secret@default/test-secret/app_key]
-  override:
-    nodeAgent:
-      env:
-        - name: DD_SECRET_BACKEND_COMMAND
-          value: "/readsecret_multiple_providers.sh"
 ```
 
 **Remarks:**
 
-* For the "Agent" and "Cluster Agent", others options exist to configure secret backend command:
+* The `"/readsecret_multiple_providers.sh"` helper enables the Agent to directly read Kubernetes Secrets across both its own and other namespaces. Ensure that the associated ServiceAccount has the necessary permissions by assigning the appropriate Roles and RoleBindings, which can be set manually or using the following options:
+    * `global.secretBackend.enableGlobalPermissions`: Determines if a ClusterRole is created that enables the Agents to read **all** Kubernetes Secrets.
+      ```yaml
+      apiVersion: datadoghq.com/v2alpha1
+      kind: DatadogAgent
+      metadata:
+        name: datadog
+      spec:
+        global:
+          secretBackend:
+            command: "/readsecret_multiple_providers.sh"
+            enableGlobalPermissions: true
+      # ...
+      ```
+    * `global.secretBackend.roles`: Replaces `enableGlobalPermissions`, detailing the list of namespaces/secrets to which the Agents should have access.
+      ```yaml
+      apiVersion: datadoghq.com/v2alpha1
+      kind: DatadogAgent
+      metadata:
+        name: datadog
+      spec:
+        global:
+          secretBackend:
+            command: "/readsecret_multiple_providers.sh"
+            roles:
+            - namespace: rabbitmq-system
+              secrets:
+              - "rabbitmqcluster-sample-default-user"
+      # ...
+      ```
+      In this example, a Role is created granting read access to the secret `rabbitmqcluster-sample-default-user` in the `rabbitmq-system` namespace.
+
+      **Note**: Each namespace in the list must be included in the DatadogAgent controller by setting `WATCH_NAMESPACE` or `DD_AGENT_WATCH_NAMESPACE` environment variables on the **Datadog Operator** container.
+
 
-  * **DD_SECRET_BACKEND_ARGUMENTS**: those arguments will be specified to the command when the agent executes the secret backend command.
-  * **DD_SECRET_BACKEND_OUTPUT_MAX_SIZE**: maximum output size of the secret backend command. The default value is 1048576 (1Mb).
-  * **DD_SECRET_BACKEND_TIMEOUT**: secret backend execution timeout in second. The default value is 5 seconds.
+* For the Agent and Cluster Agent, others options exist to configure the secret backend command:
+  * `global.secretBackend.args`: These arguments are supplied to the command when the Agent executes the secret backend command.
+  * `global.secretBackend.timeout`: secret backend execution timeout in second. The default value is 30 seconds.
+* For versions prior to Operator 1.11+, `spec.global.secretBackend` is unavailable. You should follow [these instructions][3] instead.
```

Comment:
```suggestion
* For the Agent and Cluster Agent, there are other configuration options for the secret backend command:
  * `global.secretBackend.args`: these arguments are supplied to the command when the Agent executes the secret backend command.
  * `global.secretBackend.timeout`: secret backend execution timeout in seconds. The default value is 30 seconds.
* For versions prior to Operator 1.11, `spec.global.secretBackend` is unavailable. You should follow [these instructions][3] instead.
```

Did you consider keeping the <1.11 instructions in this doc as a separate section?

Reply:
> Did you consider keeping the <1.11 instructions in this doc as a separate section?

Yes but felt it would have been this guide even longer than it is currently, so preferred to link instead of a section